docs/views/layout_unbranded.html

```diff
@@ -1,24 +1,13 @@
-{#- We can't mount GOV.UK Frontend's assets at root as it's done automatically by the extensions framework. -#}
-{%- set assetPath = '/govuk/assets' -%}
-
-{% extends "govuk/template.njk" %}
+{% extends "layout.html" %}
 
 {% block headIcons %}
   <link rel="shortcut icon" href="{{ asset_path }}images/unbranded.ico?0.18.3" type="image/x-icon" />
-  <link rel="mask-icon" href="{{ asset_path }}images/gov.uk_logotype_crown.svg?0.18.3" color="#0b0c0c">
```

Comment:
The icons are still added when using `layout.html` as they are used by GOV.UK Design System template. Could we use `{{ super() }}` to add them here too?

Reply:
For the unbranded template we want to intentionally avoid the default icons as they're branded, so I don't think we want to use super here.

Reply:
Okay that makes sense 👍